plugins/ocm-backend/package.json

```diff
@@ -42,20 +42,23 @@
     "prepack": "backstage-cli package prepack",
     "start": "backstage-cli package start",
     "test": "backstage-cli package test --passWithNoTests --coverage",
-    "tsc": "tsc"
+    "tsc": "tsc",
+    "build:api-docs": "./scripts/openapi-doc.sh",
+    "build:api-server": "yarn backstage-repo-tools package schema openapi generate --server"
   },
   "configSchema": "config.d.ts",
   "dependencies": {
     "@backstage/backend-common": "^0.23.3",
+    "@backstage/backend-dynamic-feature-service": "^0.2.15",
+    "@backstage/backend-openapi-utils": "^0.1.15",
```

Comment:
Sorry, I think I misunderstood you earlier. You were talking about these [peer dependencies](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/ocm-backend/dist-dynamic/package.json#L67), right? Yes, I believe that they need to be added there as well.

Reply:
Yes. I will update.